app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+         displayed in the settings dialog and is used to label an ON/OFF switch which toggles whether
+         an application crash is reported with out prompting the user first. -->
     <string name="settings_crash_reporting">Crash Reporting</string>
+    <!-- LOCALIZATION NOTE (settings_telemetry): The string 'settings_telemetry' is displayed in the
+         settings dialog and is used to label an ON/OFF switch which toggles whether telemetry is
+         reported back to Mozilla. -->
     <string name="settings_telemetry">Telemetry</string>
+    <!-- LOCALIZATION NOTE (settings_privacy): The string 'settings_privacy' is used to label a button
+         that when pressed loads a web page the describes Mozilla's privacy policy. -->
     <string name="settings_privacy">Privacy Policy</string>
+    <!-- LOCALIZATION NOTE (settings_report_issue): The string 'settings_report_issue' is used to label
+         a button that when pressed loads a web page that allows the user file an issue or problem
+         they may have encountered while using the application. -->
     <string name="settings_report_issue">Report an issue</string>
+    <!-- LOCALIZATION NOTE (settings_version_developer): FIXME This string probably doesn't need to be localized/ -->
     <string name="settings_version_developer">Developer Build</string>
+    <!-- LOCALIZATION NOTE (settings_developer): The string 'settings_developer' is used to label a
+         button in the settings dialog that when pressed opens a dialog box that contains setting
+         an application or web developer might want to change. -->
     <string name="settings_developer">Developer</string>
+    <!-- LOCALIZATION NOTE (settings_options): FIXME this string is part of 'settings_developer' and is broken up to make it fit. -->
     <string name="settings_options">Options</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_restart): The string 'restart_dialog_restart' is the title of
+         a dialog box shown to the user when a settings change requires the application to restart. -->
     <string name="restart_dialog_restart">Restart Required</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_text): The string 'restart_dialog_text' is the text displayed
+         in the restart dialog box explaining why the restart is required. -->
     <string name="restart_dialog_text">You must restart the Firefox Reality app in order to complete changes. Would you like to do that now?</string>
+    <!-- LOCALIZATION NOTE (restart_dialog_button):  The string 'restart_dialog_button' is used to
+         label the button the user presses to restart the application in the restart dialog. -->
     <string name="restart_dialog_button">Restart Now</string>
+    <!-- LOCALIZATION NOTE (developer_options_remote_debugging): The string 'developer_options_remote_debugging'
+         labels an ON/OFF switch in the developer options dialog and is used to toggle remote debugging
```

Comment:
do you want to capitalise `developer options` -> `Developer Options` here (and elsewhere)?

Reply:
you can probably change `remote debugging` -> `Remote Debugging` too as well